<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuneEditMainWindow</class>
 <widget class="QMainWindow" name="RuneEditMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RuneEditMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Edler Furthark</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Younger Furthark</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QPushButton" name="e">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>e</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="i">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>i</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="q">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>q</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="d">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>d</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="o">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>o</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="u">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>u</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="a">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="f">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>f</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="h">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>h</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="j">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>j</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QPushButton" name="l">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>l</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="x">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="v">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>v</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="y">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="n">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>n</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="b">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="c">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>c</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="r">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>r</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="g">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QPushButton" name="k">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>k</string>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QPushButton" name="p">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>p</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="z">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="t">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>t</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="s">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="w">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>w</string>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="m_showLatin">
      <property name="text">
       <string>Show Latin</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="m_editText"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save </string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
